README.md

```diff
@@ -1,9 +1,115 @@
-# JupiterOne Starbase
+<h1 align="center">
+  <img src="./img/branding/logo-light-bg.png" alt="Starbase Logo" width="250" /></br></br>
+  <strong style="font-size:75px;">Democratizing graph-based security analysis 🚀</strong>
+</h1></br>
 
-Starbase by JupiterOne is an orchestrator application for running graph
-ingestion projects and optionally pushing them to a storage endpoint.
+Starbase from [JupiterOne](https://jupiterone.com), collects assets and relationships from
+services and systems including cloud infrastructure, SaaS applications, security controls,
+and more into an intuitive graph view backed by the [Neo4j](https://neo4j.com/) database.
 
-## Development
+**Security is a basic right**. Starbase's goal is to **democratize graph-based security analysis** and 
+overall visibility into external services and systems. Our team believes that in order to secure any 
+system or service, you must have:
+
+- **Knowledge** of the assets that you have
+- **Knowledge** of the relationships between assets that you have
+- **Knowledge** of what questions to ask about what you have
+
+<p align="center">
+  <img src="./img/starbase-demo.gif" alt="Starbase Demo"/>
+</p>
+
+## Why Starbase?
+
+Starbase offers three key advantages:
+
+1. **Depth and breadth** - Deep visibility from a [breadth of external services and systems](#available-integrations--connectors). Thousands of entities (verticies) and relationships (edges) are available out-of-the-box.
+2. **Uniform data model** - The data that Starbase collects is _automatically_ classified, making it easy to develop _generic_ queries.
+3. **Easily extensible** - Starbase graph integrations can be easily developed!
+
+## Available Integrations / Connectors
+
+Starbase supports [70+](https://github.com/jupiterone?q=graph-&type=all&language=&sort=)
+open source graph integrations!
+
+Here are some highlights:
+
+- [Azure](https://github.com/jupiterone/graph-azure)
+- [Bitbucket](https://github.com/jupiterone/graph-bitbucket)
+- [GitHub](https://github.com/jupiterone/graph-github)
+- [Google Cloud](https://github.com/jupiterone/graph-google-cloud)
+- [Google Workspace](https://github.com/jupiterone/graph-google)
+- [Jira](https://github.com/jupiterone/graph-jira)
+- ...
+
+<details>
+  <summary><b>❗Click here to expand a full list of supported graph integrations❗</b></summary>
```

Comment:
How is this list going to be maintained?

Reply:
I thought that same thing. Haven't worked that out yet 😬 . Constant list is easiest for now.